jaq/src/main.rs

```diff
@@ -456,6 +459,11 @@ where
 
 fn fmt_val(f: &mut Formatter, opts: &PpOpts, level: usize, v: &Val) -> fmt::Result {
     use yansi::Paint;
+    if opts.img_auto && matches!(v, Val::Str(_)) && is_image(v) {
+        writeln!(f, "key:")?;
```

Comment:
Why the "key:"?

Reply:
100% agree! It's left over after experimenting in the evening and later `git rebase -i` clearning up/squashing commits and overlooking some experimental left over.
Thank you for thorough review!

Reply:
* [x] Removed "key" in like 463 , here https://github.com/VariousForks/jaq/commit/ea63f7977652aa311dee9974924a635a7e9dd401#diff-40e9345b37cac1978aae751353ce0b185eaecb9330f443e4b40bff82f80c2e16R463-R470